code/modules/clothing/suits/armor.dm

```diff
@@ -182,6 +182,38 @@
 		return 1
 	return 0
 
+/obj/item/clothing/suit/armor/reactive/teleport
```

Comment:
I suppose that makes sense. Though admins might want to be able to spawn in a safe version, and they wouldn't be able to do that by just varediting it.